src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/jpa/repository/ResettlementAssessmentRepository.kt

```diff
@@ -97,11 +97,10 @@ interface ResettlementAssessmentRepository : JpaRepository<ResettlementAssessmen
       creationDate, 
       submissionDate, 
       createdBy, 
-      caseNoteText  
+      coalesce(caseNoteText, 'Resettlement status set to: SUBMITTED')  
```

Comment:
This should be the pathway status (from the `status_changed_to` column) not the report status